www/components/Layout.tsx

```diff
@@ -1,36 +1,40 @@
 import React, { ReactNode } from "react";
 import { createTheme, WuiProvider } from "@welcome-ui/core";
 import { darkTheme } from "@welcome-ui/themes.dark";
-import { welcomeTheme } from "@welcome-ui/themes.welcome";
 import { Flex } from "@welcome-ui/flex";
 import { Stack } from "@welcome-ui/stack";
 import { Text } from "@welcome-ui/text";
 import { Box } from "@welcome-ui/box";
 import { Pkg } from "../lib/docs";
 import styled from "styled-components";
 
-const Main = styled(Box)`
-  min-height: 100vh;
-`;
-
 const theme = createTheme(darkTheme);
 
-export default function Layout({
-  title,
-  children,
-}: {
+interface Props {
```

Comment:
teach me senpai, what this better?

Reply:
Assuming s/what/why/.

It's mainly readability. To me the version with the interface inline is way harder to read; the same fields repeat with only `}: {` separating them, and it's hard for me to visually parse where one block starts and the other ends, especially when there are >3 props.

Another reason this is a good practice is it enables easily reusing the props specification if needed elsewhere. Like, if you want to wrap / extend the component, that's easier if you can refer to its props type explicitly.

Reply:
... In an ideal world I'd want to write `function component({field: bool, otherField: string})` but TypeScript is not ideal that way :D